test/integration/apiserver/oidc/oidc_test.go

```diff
@@ -943,6 +947,7 @@ jwt:
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I know newer versions of Go don't need `tt := tt` but I think we should include that until older Go versions are phased out of the ecosystem.

Reply:
done!

Reply:
yep, that's a go 1.22 change: https://tip.golang.org/doc/go1.22 🎉